namespaces/cloud-platform-live-0.k8s.integration.dsd.io/check-my-diary-dev/01-rbac.yaml

```diff
@@ -14,3 +14,23 @@ roleRef:
   kind: ClusterRole
   name: admin
   apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tiller
+  namespace: check-my-diary-dev
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: tiller
+  namespace: check-my-diary-dev
+subjects:
+- kind: ServiceAccount
+  name: tiller
+  namespace: check-my-diary-dev
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Can you please change this to `admin`? The permissions should be equivalent, but `admin` allows us to define them centrally them as we introduce new features / changes.